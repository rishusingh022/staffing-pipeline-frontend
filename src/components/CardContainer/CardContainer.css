.card-container {
  width: 70%;
  display: flex;
  align-items: center;
  justify-content: center;
  flex: 1;
}
