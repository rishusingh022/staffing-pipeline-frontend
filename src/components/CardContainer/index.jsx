import * as React from 'react';
import './CardContainer.css';
import PropTypes from 'prop-types';
const CardContainer = props => {
  return <div className="card-container">{props.children}</div>;
};
export default CardContainer;

CardContainer.propTypes = {
  children: PropTypes.element.isRequired,
};
