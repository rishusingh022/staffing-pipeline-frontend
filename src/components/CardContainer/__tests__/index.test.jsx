import { render } from '@testing-library/react';
import * as React from 'react';
import CardContainer from '..';

describe('CardContainer', () => {
  it('should render without crashing', () => {
    const { asFragment } = render(
      <CardContainer>
        <div>Test</div>
      </CardContainer>
    );
    expect(asFragment()).toMatchSnapshot();
  });
  it('Should contain a div with text "card-container"', () => {
    const { getByText } = render(
      <CardContainer>
        <div>card-container</div>
      </CardContainer>
    );
    expect(getByText('card-container')).toBeTruthy();
  });
});
