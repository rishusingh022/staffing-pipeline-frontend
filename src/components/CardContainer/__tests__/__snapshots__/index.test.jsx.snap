// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer should render without crashing 1`] = `
<DocumentFragment>
  <div
    class="card-container"
  >
    <div>
      Test
    </div>
  </div>
</DocumentFragment>
`;
